.github/workflows/reusable-release.yaml

```diff
@@ -95,6 +103,16 @@ jobs:
         run: |
           rm gpg.key
 
+      # Size of `dist` dir for release is large.
+      # We need to remove unneeded binaries and archives
+      # to avoid cases when cache will be removed due to size
+      - name: Remove unneeded binaries and archives
+        if: ${{ inputs.goreleaser_config == 'goreleaser.yml' }}
+        run: |
+          rm -r dist/build-*
+          rm -r dist/*tar.gz*
+          rm -r dist/*zip
```

Comment:
Can we remove all files other than `.deb` and `.rpm`? And after using `easimon/maximize-build-space`, there is enough space. We may not need to remove these files.

```
find . -type f ! \( -name "*.rpm" -o -name "*.deb" \) -exec rm -f {} +
```

Reply:
what about `*.dep.pem` or `*.dep.sig` files? 
i thought deploy scripts use them.

Reply:
I'm not sure, but easimon/maximize-build-space gives us enough space. We can remove this step for now.